CHANGELOG.rst

```diff
@@ -3,6 +3,13 @@
 salt-formula
 ============
 
+0.0.4
```

Comment:
You should also create and push git tag (best GPG signed) if you wish to make a new version. Also update `d/changelog` accordingly.